apps/api/src/app/integrations/usecases/get-novu-integration/get-novu-integration.command.ts

```diff
@@ -0,0 +1,8 @@
+import { EnvironmentCommand } from '../../../shared/commands/project.command';
+import { IsString } from 'class-validator';
+import { ChannelTypeEnum } from '@novu/shared';
+
+export class GetNovuIntegrationCommand extends EnvironmentCommand {
+  @IsString()
```

Comment:
```suggestion
  @IsEnum(ChannelTypeEnum)
```

Could be further validated with the `IsEnum` validator for proper error handling